src/rust/engine/src/lib.rs

```diff
@@ -871,6 +875,14 @@ pub extern "C" fn flush_log() {
   LOGGER.flush();
 }
 
+#[no_mangle]
+pub extern "C" fn override_thread_logging_destination(new_destination: u8) {
```

Comment:
Do you anticipate this being used to actually set both bit flags... ie: "write to both"? If not, you can use the enum directly in the API like so: https://github.com/pantsbuild/pants/blob/1ad981f516f16f4b73df0bb5631d444ff838fe19/src/python/pants/engine/native.py#L419-L425

Reply:
+1 to the curiosity about this.

Reply:
Switched to exclusive. Went down a bit of a cbindgen rabbit-hole to get there.